src/main/java/org/schemaspy/output/xml/dom/XmlColumnFormatter.java

```diff
@@ -74,6 +76,9 @@ public Node appendColumn(Node tableNode, TableColumn column) {
         DOMUtil.appendAttribute(columnNode, "id", String.valueOf(column.getId()));
         DOMUtil.appendAttribute(columnNode, "name", column.getName());
         DOMUtil.appendAttribute(columnNode, "type", column.getTypeName());
+        DOMUtil.appendAttribute(columnNode, "typeCode", String.valueOf(column.getType() == null ?
+                DEFAULT_JDBC_TYPE_CODE : column.getType()));
+        DOMUtil.appendAttribute(columnNode, "defaultValue", String.valueOf(column.getDefaultValue()));
```

Comment:
Just for your information.
#488 
We have an issue that, we might report string "null" when it's an actuall null.

Reply:
i see yes string.valueof does that

Reply:
defaultValue is now added twice.

https://github.com/schemaspy/schemaspy/blob/7595bc318bcb280629432052c21136e206e44051/src/main/java/org/schemaspy/output/xml/dom/XmlColumnFormatter.java#L81

https://github.com/schemaspy/schemaspy/blob/7595bc318bcb280629432052c21136e206e44051/src/main/java/org/schemaspy/output/xml/dom/XmlColumnFormatter.java#L86-L95